### ISSUE TITLE
Mesh_2 segfault

### ISSUE THREAD

**Issue by sloriot** [CGAL/cgal#2456]
I have an old file that is making mesh_2 crashing in ipe but also in the mesh_2 demo (IIRC).
For the record I'm putting the files here. It might be a duplicate of #980
[mesh_2_br.zip](https://github.com/CGAL/cgal/files/1323914/mesh_2_br.zip)

**Comment by lrineau** [comment]
#980 is an issue about the I/O of "CDT_plus_2". It cannot be the same error.

**Comment by lrineau** [comment]
I was able to reduce the data set to the following `.poly` file:
```
23 2 0 0

1 0 0
2 320 0
3 320 16
4 320 32
5 320 48
6 320 64
7 320 80
8 320 96
9 320 112
10 320 128
11 304 128
12 288 128
13 272 128
14 256 128
15 240 128
16 224 128
17 208 128
18 192 128
19 176 128
20 160 128
21 144 128
22 128 128
23 112 128

6 0
4 4 1
35 1 19
37 1 20
38 21 20
39 1 21
40 22 21

0
```
to be used with `demo/Mesh_2/mesh.cpp`.

**Comment by lrineau** [comment]
I have a fix in #2467.
